### PR TITLE
[TAS-37] Bugfix: mktemp error when building on macOS and pako package lost problem.

### PR DESCRIPTION
**Pull request by RayKr**
fix: 
```
mktemp: too few X's in template ‘tmp’
```
fix: 
```
symlink has no referent: "/Users/jzl/code/zotero/build/resource/pako.js"
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1356) [sender=3.2.7]

Traceback (most recent call last):
  File "/Users/jzl/code/zotero/app/scripts/prepare_build", line 75, in main
    subprocess.check_call([
  File "/Users/jzl/miniforge3/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['rsync', '-aL', '--exclude', '.*', '--exclude', '#*', '--exclude', 'package.json', '--exclude', 'package-lock.json', './', '/Users/jzl/code/zotero/app/tmp/zotero/']' returned non-zero exit status 23.
```

### CODE REVIEW

File: app/config.sh [hunk 1]
```diff
@@ -54,7 +54,7 @@ BUILD_PLATFORMS=""
 NUM_INCREMENTALS=6
 
 if [ "`uname`" = "Darwin" ]; then
-	alias mktemp='mktemp -t tmp'
+	alias mktemp='mktemp -t tmp.XXXXXXXXXX'
```
Comment (dstillman) [inline app/config.sh:57]: Where exactly are you getting an error? This code only runs on macOS, and at least on current versions of macOS `-t` is for a prefix, so `-t tmp` is correct. It's also seemingly unnecessary, though — without `-t tmp` `-d` defaults to `-t tmp`, so this line could just be removed.

**Comment by dstillman** [comment]
What is `TAS-37`?

**Comment by RayKr** [comment]
never mind. that is a my tip of task ：）

File: app/config.sh (hunk 1 above)
Comment (RayKr) [reply app/config.sh:57]: It really can't work only with `-t tmp` on my macOS (Apple Silicon M1, MacBook Pro, Ventura 13.5.1)
image
So this line could be removed.

Comment (dstillman) [reply app/config.sh:57]: We can remove the line, but just to be clear, you've installed the GNU mktemp. This doesn't happen with the bundled BSD mktemp.

Comment (dstillman) [reply app/config.sh:57]: I've removed the line in https://github.com/zotero/zotero/commit/e1def9971c18999b0e0bb127fa3f2cb1ccd59d5f. You can undo this change.

Comment (RayKr) [reply app/config.sh:57]: Okey, I'm clear. Thanks:)

**Comment by dstillman** [comment]
Thanks!

**Pull request merged**